src/utils/databaseUtil.ts

```diff
@@ -1,15 +1,15 @@
-import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+import fs from 'fs';
```

Comment:
To clarify: With ES imports there are named imports/exports and default imports/exports. As a separate concept there's grouping multiple related functions in an object, which effectively puts them under the same name.

For our code we prefer a named export that points to several related functions (like in our utils). Outside of our code (like in node) it is common to use the default export.

Importing external libraries under a single name makes it easier to inspect what is available (like typing fs. and getting autocomplete) and more importantly it lets us stub functions off of the name for tests.